README.md

```diff
@@ -16,5 +16,5 @@ This package contains the following daemon(s):
 
 ## Installation Notes
 1. Download and install the [Vimba X SDK](https://www.alliedvision.com/en/products/software/vimba-x-sdk/) from the Allied Vision web page.
-2. Install the `vmbpy` package (not to be confused with [VimbaPython](https://github.com/alliedvision/VimbaPython), which is an older API). At the time of this writing, vmbpy was not available on Github, conda, or pip (though documentation suggests Allied Vision intends to host it on Github).  The package wheel can be found in the Vimba X program directory. 
+2. Install the `vmbpy` package (not to be confused with [VimbaPython](https://github.com/alliedvision/VimbaPython), which is an older API). At the time of this writing, vmbpy was not available on Github, conda, or pip (though documentation suggests Allied Vision intends to host it on Github).  The package wheel can be found in the Vimba X program directory.
```

Comment:
Those are the same thing, as far as I can tell. The GitHub repo is up to date and contains the same code.

Reply:
There is a closed issue stating that they do not intend to place it on pypi, which is regrettable, but yeah... there is also an older project on pypi, pymba, which was providing similar bindings, though I think they have stopped since official support started

Reply:
sadly, they are not.  vmbpy is the supposed new API that VimbaPython can migrate to.  SDK has the documentation for vmbpy (and the source), and I can verify they are different than the VimbaPython package.  

Not sure why vmbpy is not on Github though--the documentation actually has links to github repository pages that do not exist (or perhaps they are private atm?)

Reply:
vmbpy is licensed under BSD-2, which I believe means that I can copy/paste that repo in here, if we were so inclined...

Reply:
wow, those are very similar packages...

Reply:
yes`*`

`*` it still relies on non-free C libraries (which... I may add they give you the source for... just don't let you _actually_ distribute them? their license agreement is a little murky on that point, actually)